ui/pages/institutional/confirm-connect-custodian-modal/confirm-connect-custodian-modal.js

```diff
@@ -15,9 +15,9 @@ import {
   Text,
   Modal,
   ModalOverlay,
-  ModalContent,
-  ModalHeader,
 } from '../../../components/component-library';
+import { ModalContent } from '../../../components/component-library/modal-content/deprecated';
+import { ModalHeader } from '../../../components/component-library/modal-header/deprecated';
 
 const ConfirmConnectCustodianModal = ({
   onModalClose,
```

Comment:
Screenshots showing no visual changes to UI

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 5 01 41 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/d210a8e4-a7c5-4aaa-a6cb-aef8c8c8cf31">

### After

<img width="1512" alt="Screenshot 2023-12-10 at 5 01 50 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/ec5fdfa7-6379-4fac-93ad-4525cce208fd">